### ISSUE TITLE
illegal hardware instruction (core dumped)

### ISSUE THREAD

**Issue by leshow** [phildawes/racer#145]
```
~ ❯❯❯ racer complete std::co                                                   ⏎
racer: symbol lookup error: racer: undefined symbol: _ZN9LOG_LEVEL20h6bc9b4a0f1800ebdqiaE

```
I downloaded the rust-nightly-src and unpacked it and set my RUST_SRC_PATH to it's src dir

```
~ ❯❯❯ export -p RUST_SRC_PATH                                                  ⏎
typeset -x RUST_SRC_PATH=/home/leshow/Development/rust/rust-bin/rustc-nightly/src/


```

I've no idea why racer is failing.

**Comment by ebfe** [comment]
Building without the `--release` flag might help. See #139.

**Comment by leshow** [comment]
sorry i tried the rust-nightly src and i got a different error, please see edited comment

**Comment by ebfe** [comment]
That error looks like you updated rust but did not recompile racer. Since  racer depends on shared libraries from the rust installation, you have to recompile it everytime you upgrade rust.

**Comment by leshow** [comment]
in that case, racer seems to fail to build

```
==> Starting build()...
src/racer/ast.rs:80:67: 80:72 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:80                         self.ident = Some(token::get_ident(ident).get().to_string());
                                                                                      ^~~~~
src/racer/ast.rs:87:71: 87:76 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:87                                     let name = token::get_ident(name).get().to_string();
                                                                                          ^~~~~
src/racer/ast.rs:358:53: 358:58 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:358         let name = token::get_ident(seg.identifier).get().to_string();
                                                                         ^~~~~
src/racer/ast.rs:477:70: 477:75 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:477                 let methodname = token::get_ident(spannedident.node).get().to_string();
                                                                                          ^~~~~
src/racer/ast.rs:506:69: 506:74 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:506                 let fieldname = token::get_ident(spannedident.node).get().to_string();
                                                                                         ^~~~~
src/racer/ast.rs:627:72: 627:77 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:627                     let name = String::from_str(token::get_ident(name).get());
                                                                                            ^~~~~
src/racer/ast.rs:649:63: 649:68 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:649                 self.name = Some(token::get_ident(item.ident).get().to_string());
                                                                                   ^~~~~
src/racer/ast.rs:685:63: 685:68 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:685                 self.name = Some(token::get_ident(item.ident).get().to_string());
                                                                                   ^~~~~
src/racer/ast.rs:736:80: 736:85 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:736                 self.name = Some(String::from_str(token::get_ident(item.ident).get()));
                                                                                                    ^~~~~
src/racer/ast.rs:752:80: 752:85 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:752                 self.name = Some(String::from_str(token::get_ident(item.ident).get()));
                                                                                                    ^~~~~
src/racer/ast.rs:754:47: 754:52 error: type `&syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:754                     self.realname = Some(istr.get().to_string());
                                                                   ^~~~~
src/racer/ast.rs:771:80: 771:85 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:771             self.generic_args.push(String::from_str(token::get_ident(ty.ident).get()));
                                                                                                    ^~~~~
src/racer/ast.rs:784:80: 784:85 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:784             self.generic_args.push(String::from_str(token::get_ident(ty.ident).get()));
                                                                                                    ^~~~~
src/racer/ast.rs:791:62: 791:67 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:791         let name = String::from_str(token::get_ident(_ident).get());
                                                                                  ^~~~~
src/racer/ast.rs:805:72: 805:77 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:805                 self.name = String::from_str(token::get_ident(i.ident).get());
                                                                                            ^~~~~
src/racer/ast.rs:815:92: 815:97 error: type `syntax::parse::token::InternedString` does not implement any method in scope named `get`
src/racer/ast.rs:815                     self.values.push((String::from_str(token::get_ident(variant.node.name).get()), point as usize));
                                                                                                                ^~~~~
```

**Comment by ebfe** [comment]
This should be fixed with #142. But it isn't merged yet.

**Comment by leshow** [comment]
so if i build again (https://aur.archlinux.org/packages/racer-git/) all should be well?
